src/protocols/secure_channel/CASESession.cpp

```diff
@@ -419,8 +403,10 @@ CHIP_ERROR CASESession::HandleSigmaR1(System::PacketBufferHandle & msg)
 
     {
         const ByteSpan * ipkListSpan = GetIPKList();
```

Comment:
Even though we are not yet able to use the Identity Protection group for actual IPK distribution, it feels like it would be more intuitive and slightly closer to the spec if the IPKList (even this canned IPK list with only one all-zero entry) was stored in `FabricInfo`, as this is inherently fabric-scoped.

I just mention this because this code is complicated and the closer it is to the spec, the easier it will be to understand.

Reply:
Yes, it makes sense to move it to FabricTable.
More changes are coming to FabricTable code, so I would prefer to do it as part of that in a separate PR.